### DISCUSSION TITLE
next filtered record id on edit page

### DISCUSSION THREAD

**Discussion by aurawindsurfing** [filamentphp/filament#6147] [General]
Hi!

Any idea how would you get next record id on edit page of the resource, while filter are applied on the table?

Obviously my example is wrong:

```
            Action::make('next')
                ->url(fn () => $this->getResource()::getUrl('edit', ['record' => ($this->record->id + 1)])),
```

Thanks
@danharrin

**Comment by danharrin** [comment]
I can't think of a solution for this, sorry
